src/pages/Applicant/List/Applicants/container.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { FunctionComponent } from "react";
+import { useQuery } from "@apollo/react-hooks";
+import { useHistory } from "react-router-dom";
+import NotFound from "$pages/NotFound";
+import { GET_APPLICANTS, getTranslations } from "$queries";
+import Loading from "$pages/Loading";
+import { Applicants } from "./component";
+import { RoutesBuilder } from "$src/routesBuilder";
+
+const ApplicantsContainer: FunctionComponent = () => {
+  const history = useHistory();
+  // TODO: Agregar las translation "view" en el back
+  const translationsResponse = useQuery(getTranslations, {
+      variables: { paths: ["edit"] }
+    }
+  );
+  const { data, error, loading } = useQuery(GET_APPLICANTS);
+  if (error || translationsResponse.error) return <NotFound/>;
```

Comment:
ese error puede no ser de not found... 🤔 
en particular el de translations

Reply:
claro, probablemente necesitemos un 500

Reply:
si, un 500 o un 400 si es que se hizo una bad request

Reply:
se resuelve en este issue: https://fiuba-laboral-v2.atlassian.net/browse/TPP-101